src/graph/graph.photos.ts

```diff
@@ -6,9 +6,12 @@
  */
 
 import { ResponseType } from '@microsoft/microsoft-graph-client';
+import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+import { IDynamicPerson } from '../components/mgt-person/mgt-person';
```

Comment:
We should factor out `IDynamicPerson` to a seperate file so the graph modules don't depend on a component module